src/Connect.js

```diff
@@ -111,13 +115,7 @@ class Connect {
       requestAddress: () => {
         const requestID = 'addressReqProvider'
         this.requestDisclosure({accountType: this.accountType || 'keypair'}, requestID)
-        return this.onResponse(requestID).then(payload => {
-          // improve set/get state
-          this.address = payload.res.address
-          this.mnid = payload.res.mnid
-          this.did = payload.res.did
-          return this.address
-        })
+        return this.onResponse(requestID).then(payload => this.address)
```

Comment:
if kept this way, would be payload.address, although we still want to likely change to set state on connect object from provider